### ISSUE TITLE
synpress.json not clear to beginner

### ISSUE THREAD

**Issue by yvesbou** [Synthetixio/synpress#425]
Hey I have some questions around the synpress.json configuration. I started to use synpress. I followed this [tutorial](https://medium.com/andamp/how-to-setup-synpress-for-wen3-dapp-frontend-test-automation-with-metamask-73396896684a), which is really great! But it didn't explain everything I needed. 

I repeatedly got the following error

`Can't run because no spec files were found.`

I thought that this part of the synpress.json is important, as testFiles were listed..

```
"component": {
    "componentFolder": ".",
    "testFiles": "**/*spec.{js,jsx,ts,tsx}"
  },
```

However, it turned out, that the integrations folder `"integrationFolder": "integration",` didn't match the default cypress project setup. `cypress/integration/..`

well fair enough, however it runs every test file inside the integrations folder. 
How can I restrict that? I tried retricting via the key-value pair of "testFiles". I thought it was the component key inside the `synpress.json` ...

So to summarise, I don't understand the difference between 

```
"component": {
    "componentFolder": ".",
    "testFiles": "**/*spec.{js,jsx,ts,tsx}"
  },
```

and 

`"integrationFolder": "integration",`

<img width="775" alt="Screenshot 2022-05-31 at 17 00 25" src="https://user-images.githubusercontent.com/63062257/171205401-66a28348-e784-435a-a667-7a32171d9618.png">

thank you for your help! :)

**Comment by drptbl** [comment]
Hey @yvesbou, this is old and probably not relevant to latest synpress version (it's `.js` config now, not `.json`). If this issue still persists, please re-open. Thanks!
